apps/app/src/utils/pageScroll.ts

```diff
@@ -1,20 +1,30 @@
 import "client-only";
 
-export const lockScroll = ({ mobileOnly }: { mobileOnly: boolean }) => {
-	document.body.style.paddingRight = `${window.innerWidth - document.body.offsetWidth}px`;
+type PropsType = {
+	mobileOnly: boolean;
+};
```

Comment:
Po co?

Reply:
Wcześniej w typie `PropsType` było więcej propsów, a teraz uznałem, że skoro wykorzystuje w 2 fukcjach takie same propsy to postanowiłem to pozostawić. Rozumiem że lepiej wrócić do:
```ts
export const lockScroll = ({ mobileOnly }: { mobileOnly: boolean }) => {
```